week-07/day-4/DI/DI/Views/Home/Email.cshtml

```diff
@@ -0,0 +1,18 @@
+﻿
+@model List<string>
+
+<html xmlns="http://www.w3.org/1999/xhtml">
+    <head>
+        <title>Email</title>
+    </head>
+    <body>
+        <style>
+            body {
+                color: @Model[1];
```

Comment:
for better clarity, these could be put into spearate variables at the top of the view

```cs
@{
  string color = Model[1];
  string colorName = Model[0];
}
```